Sources/TensorFlow/Loss.swift

```diff
@@ -52,7 +52,7 @@ public func meanAbsoluteError<Scalar: TensorFlowFloatingPoint>(
     predicted: Tensor<Scalar>,
     expected: Tensor<Scalar>
 ) -> Tensor<Scalar> {
-    l1Loss(predicted: predicted, expected: expected, reduction: { $0.mean() })
```

Comment:
I believe there is also a couple of loss functions that use `sum()`. Shouldn't we modify those too?

Reply:
The PR tests integrating TensorFlow directly into a toolchain:
* Delete `TensorFlow` module and artifacts from an existing toolchain.
* Use toolchain to build `swift-apis`.
* Copy built `TensorFlow` artifacts back into the toolchain.

`_mean` is a workaround for linker errors encountered during CI for `swift-models`:
```
<compiler-generated>:0: error: undefined reference to 'AD__$s10TensorFlow19softmaxCrossEntropy6logits6labels9reductionAA0A0VyxGAH_AGys5Int32VGA2HXFtAA0aB13FloatingPointRzlFfA1_A2HcfU___jvp_src_0_wrt_0'
<compiler-generated>:0: error: undefined reference to 'AD__$s10TensorFlow19softmaxCrossEntropy6logits6labels9reductionAA0A0VyxGAH_AGys5Int32VGA2HXFtAA0aB13FloatingPointRzlFfA1_A2HcfU___vjp_src_0_wrt_0'
<compiler-generated>:0: error: undefined reference to 'AD__$s10TensorFlow19softmaxCrossEntropy6logits6labels9reductionAA0A0VyxGAH_AGys5Int32VGA2HXFtAA0aB13FloatingPointRzlFfA1_A2HcfU___pullback_src_0_wrt_0'
```

Adding `_mean` and renaming usages was sufficient to fix these errors, so `_sum` is not necessary at the moment. We're debugging these linker errors so we can remove `_mean` ASAP.

Reply:
Great, thanks Dan!

Reply:
Is there a bug filed for the linker error?

Reply:
Filed [TF-690](https://bugs.swift.org/browse/TF-690].